hermes-console-vue/src/views/subscription/subscription-metadata/SubscriptionMetadata.spec.ts

```diff
@@ -1,4 +1,5 @@
 import { beforeEach } from 'vitest';
+import { dummyOwner } from '@/dummy/topic';
 import { dummySubscription } from '@/dummy/subscription';
 import { expect } from 'vitest';
 import { render } from '@/utils/test-utils';
```

Comment:
minor: we can consider adding tests for checking whether events were emitted on btn clicks